packages/devtools_app/lib/src/screens/inspector/inspector_service_polyfill.dart

```diff
@@ -19,11 +21,11 @@ final _polyfillReadyExpando = Expando<Future>();
 
 Future<void> invokeInspectorPolyfill(ObjectGroup group) {
   // Avoid invoking the polyfill more than once.
-  var polyFillReady = _polyfillReadyExpando[group.inspectorLibrary.isolate];
+  var polyFillReady = _polyfillReadyExpando[group.inspectorLibrary.isolate!];
   if (polyFillReady != null) return polyFillReady;
 
   polyFillReady = _invokeInspectorPolyfill(group);
-  _polyfillReadyExpando[group.inspectorLibrary.isolate] = polyFillReady;
+  _polyfillReadyExpando[group.inspectorLibrary.isolate!] = polyFillReady;
```

Comment:
hoist group.inspectorLibrary.isolate! into a local var so it doesn't have to be null asserted twice

Reply:
done